client/css/navStyles.css

```diff
@@ -0,0 +1,71 @@
+/* Fix top nav bar to right hand side of page */
```

Comment:
display: fixed;
top: 0;
right: 1em (or similar);

might be a better option for keeping this fixed to the top always

Reply:
I see what you mean!  I initially used this set up because it was the easiest way I could think of for keeping switch mode button separate to the register/login icons ... the way it's currently set up, "display: fixed" etc. would regroup those elements.  What if i just add "position: fixed" instead?  Would that also fix the nav bar to the top?

Reply:
You can use position: fixed on the element that contains all the nav stuff, and then justify content on the things inside?? (sorry i should have really pulled down and look but i dont know how to pull down other people's branches 😭 )